app/frontend/components/shared/permit-applications/permit-application-card.tsx

```diff
@@ -97,14 +102,25 @@ export const PermitApplicationCard = ({ permitApplication }: IPermitApplicationC
               {format(updatedAt, "MMM d, yyyy")}
             </Text>
           </Flex>
+          <Flex direction={{ base: "column", md: "row" }} gap={4}>
+            <Link href={t("permitApplication.seeBestPractices_link")} isExternal>
+              {t("permitApplication.seeBestPractices_CTA")}
+              <ArrowSquareOut></ArrowSquareOut>
+            </Link>
+            <Show above="md">
+              <Text>{"  |  "}</Text>
+            </Show>
+            <Link href={t("permitApplication.searchKnowledge_link")} isExternal>
+              {t("permitApplication.searchKnowledge_CTA")} <ArrowSquareOut></ArrowSquareOut>
+            </Link>
+          </Flex>
         </Flex>
         <Flex direction="column" align="flex-end" gap={4} flex={{ base: 0, md: 1 }} maxW={{ base: "100%", md: "30%" }}>
           <Show above="md">
             <PermitApplicationStatusTag permitApplication={permitApplication} />
             <Text>
-              <Text as="span" fontWeight={700}>
+              <Text as="p" align="right" variant="tiny_uppercase">
```

Comment:
text is always a `p` so no need for the `as` BUT also nested `p` is a no-no so should make the outer `Text` a `Flex` probably.